submodules/LIB/scripts/iob_module.py

```diff
@@ -740,12 +740,6 @@ def _copy_srcs(cls, exclude_file_list=[], highest_superclass=None):
                             ),
                         )
                         continue
-                elif directory == "hardware/common_src":
-                    shutil.copytree(
-                        os.path.join(os.getcwd(), directory),
```

Comment:
this lines is something that i added in another commit but they serve no purpose, because the directory is copied ether way in the last else. and worst maybe they are causing problems with other cores.